finsoc/app/views/main/newsletter.html.erb

```diff
@@ -1,9 +1,21 @@
+
 <div class="newsletter-container">
   <div class="title">
     <h1>NEWSLETTER</h1>
   </div>
+  <div class="dropdown">
+  <button onclick="selectNews()" class="dropbtn">Select Newsletter</button>
```

Comment:
Don't add onclick and inline JS in the html files. This behaviour should always reside in a JS file.